### DISCUSSION TITLE
How to recovery banned account?

### DISCUSSION THREAD

**Discussion by indrajithbandara** [community/community#138951] [Other Feature Feedback, Questions, & Ideas]
How to recovery GitHub banned account?

**Comment by carstenconrad1098** [comment]
I think you can contact support center.

**Comment by samueltexa** [comment]
This could help, there are several answers there,
https://github.com/orgs/community/discussions/132061

**Comment by Kremilly** [comment]
I think this is impossible. Unfortunally!
